README.md

```diff
@@ -3,6 +3,14 @@ Licensed under Creative Commons Attribution 4.0 International License
 https://creativecommons.org/licenses/by/4.0/
 -->
 
+# DEPRECATION NOTICE
+
+**NOTE**: This project is currently in `deprecated` state. Depending on your needs, please refer to one of the following projects instead:
+
```

Comment:
The closest project matching privacy-preserving off-chain smart contract execution following similar principles is in labs https://github.com/hyperledger-labs/private-data-objects